cogs/utils/botdata.py

```diff
@@ -251,7 +265,8 @@ def __init__(self):
 		self.defaults = OrderedDict([ 
 			("userinfo" , []), 
 			("guildinfo" , []), 
-			("dotapatch", None) 
+			("dotapatch", None),
+                        ("dotablog",None)
```

Comment:
This line has some spaces instead of tabs in it, could cause some issues with python i think